hudi-flink/src/main/java/org/apache/hudi/streamer/FlinkStreamerConfig.java

```diff
@@ -130,6 +131,183 @@
   @Parameter(names = {"--write-task-num"}, description = "Parallelism of tasks that do actual write, default is 4.")
   public Integer writeTaskNum = 4;
 
+  @Parameter(names = {"--partition-default-name"},
+          description = "The default partition name in case the dynamic partition column value is null/empty string")
+  public String partitionDefaultName = "__DEFAULT_PARTITION__";
+
+  @Parameter(names = {"--index-bootstrap-enabled"},
+          description = "Whether to bootstrap the index state from existing hoodie table, default false")
+  public Boolean indexBootstrapEnabled = false;
+
+  @Parameter(names = {"--index-state-ttl"}, description = "Index state ttl in days, default 1.5 day")
+  public Double indexStateTtl = 1.5D;
+
+  @Parameter(names = {"--index-global-enabled"}, description = "Whether to update index for the old partition path " +
+          "if same key record with different partition path came in, default false")
+  public Boolean indexGlobalEnabled = false;
+
+  @Parameter(names = {"--index-partition-regex"},
+          description = "Whether to load partitions in state if partition path matching， default *")
+  public String indexPartitionRegex = ".*";
+
+  @Parameter(names = {"--read-tasks-parallelism"}, description = "Parallelism of tasks that do actual read, default is 4")
+  public Integer readTasksParallelism = 4;
+
+  @Parameter(names = {"--avro-schema-path"}, description = "Avro schema file path, the parsed schema is used for deserialization")
+  public String avroSchemaPath;
+
+  @Parameter(names = {"--avro-schema"}, description = "Avro schema string, the parsed schema is used for deserialization")
+  public String avroSchema;
+
+  @Parameter(names = {"--source-query-type"}, description = "Decides how data files need to be read, in\n"
+          + "1) Snapshot mode (obtain latest view, based on row & columnar data);\n"
+          + "2) incremental mode (new data since an instantTime);\n"
+          + "3) Read Optimized mode (obtain latest view, based on columnar data)\n."
+          + "Default: snapshot")
```

Comment:
I guess we do not need these read options, because the `HoodieFlinkStreamer` only do write operation.

Reply:
removed
```
source-query-type
source-merge-type
source-hive-style-partition
read-tasks-parallelism
```